client/src/game/canvas.ts

```diff
@@ -345,12 +345,13 @@ export const createGameApp = async (
 
   const attackSprite = await prepAttackAnimation();
 
-  const showAttackAnimation = (pos: Position) => {
+  const showAttackAnimation = (pos: Position, time: number) => {
     const screenPos = mapPosToScreenPos(pos);
     attackSprite.visible = true;
     const pxPos = tileToPx(screenPos);
     attackSprite.x = pxPos.x;
     attackSprite.y = pxPos.y;
+    attackSprite.animationSpeed = ((1 / attackSprite.totalFrames) * 1) / time;
```

Comment:
love this

Reply:
oh, I just realized that it autoformatted this wrong on me! It's actually supposed to be:
(1 / attackSprite.totalFrames) * (1 / time);